### ISSUE TITLE
Authorization code (Identity Server part) is missing from the code

### ISSUE THREAD

**Issue by frank-ozgul** [dotnet/tye#829]
Hi, There is a video on youtube describing the Voting project  as a showcase for Tye tool by two speakers (Justin Kotalik and Glen Condron). In the video, they also talk about Identity Server part attached to the sample code. However, I do not see it here. If possible, could you please add that to the repo?
Thank you.

**Comment by Hammurabidoug** [comment]
Yes. Seconded. There is an IdentityServer project in the jkotalik/newVotingSample branch, but it seems to be stale. There's also an exception thrown because it expects a public IP for the identity server.

**Comment by frank-ozgul** [comment]
> Yes. Seconded. There is an IdentityServer project in the jkotalik/newVotingSample branch, but it seems to be stale. There's also an exception thrown because it expects a public IP for the identity server.

Thank you. I later noticed that actually but I have not tried  it yet.  Probably we are supposed to place our identity server's IP for the public-ip's value obtained from docker network address(see docker settings, resources)

**Comment by jkotalik** [comment]
@frank-ozgul @Hammurabidoug yeah, the public IP expectation was something we didn't expect in our demo, and had to add it to make IdentityServer work. We didn't make it a sample because of that expectation. The IP address was actually the IP address of the k8s cluster.

**Comment by Hammurabidoug** [comment]
When I saw the IdentityServer demo, I was like “ yay! They figured out how
to add dynamic OIDC redirect URIs.” But maybe I’m missing some of the magic
of Tye. Will the sample include IdentityServer? I could not figure out how
to get the Tye configuration to work with it. Otherwise I would have
submitted a PR.
